x-pack/plugins/alerting/public/hooks/use_get_maintenance_windows_list.ts

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { useQuery } from '@tanstack/react-query';
+import { useKibana } from '../utils/kibana_react';
+import { getMaintenanceWindowsList } from '../services/maintenance_windows_api/list';
+
+export const useGetMaintenanceWindowsList = () => {
```

Comment:
I think it would be nice if we can map these names to the API that they're calling (i.e. useFindMaintenanceWindow)